jfr-streaming/build.gradle.kts

```diff
@@ -0,0 +1,62 @@
+val gsonVersion: String by project
+val slf4jVersion: String by project
+val otelVersion: String by project
+val otelAlphaVersion: String by project
+
+
+plugins {
+    id("com.github.johnrengelman.shadow") version "5.2.0"
+    id("java")
+}
+
+repositories {
+    mavenCentral()
+    mavenLocal()
+}
+
+java {
+    toolchain {
+        languageVersion.set(JavaLanguageVersion.of(17))
```

Comment:
You'll still need this even with the conventions, but I'll work on upgrading our build's toolchain java to 17 (Groovy...)

Either way, add something like

https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk-extensions/jfr-events/build.gradle.kts#L15

to make sure it targets java 17